ambari-infra-manager-it/src/test/java/org/apache/ambari/infra/steps/AbstractInfraSteps.java

```diff
@@ -137,14 +137,14 @@ protected void addDocument(OffsetDateTime logtime) {
     solrInputDocument.addField("level", "INFO");
     solrInputDocument.addField("resource", "/ats/active");
     solrInputDocument.addField("ip", "172.18.0.2");
-    solrInputDocument.addField("evtTime", "2017-12-08T10:23:16.452Z");
     solrInputDocument.addField("req_caller_id", "HIVE_QUERY_ID:ambari-qa_20160317200111_223b3079-4a2d-431c-920f-6ba37ed63e9f");
     solrInputDocument.addField("repoType", 1);
     solrInputDocument.addField("enforcer", "hadoop-acl");
     solrInputDocument.addField("cliType", "rpc");
     solrInputDocument.addField("message_md5", "-6778765776916226588");
     solrInputDocument.addField("event_md5", "5627261521757462732");
     solrInputDocument.addField("logtime", new Date(logtime.toInstant().toEpochMilli()));
+    solrInputDocument.addField("evtTime", new Date(logtime.toInstant().toEpochMilli()));
```

Comment:
This seems out of place for this patch.  Did you mean to add this line?

Reply:
Yes, I changed the query text for `archive_audit_logs` to filter by `evtTime` in another patch and forgot to run the integration tests. But that patch is already merged. 
When I run the tests after changing the versions of the dependencies the tests didn't passed because `evtTime` was a constant.